src/lib/consoleArt.ts

```diff
@@ -1,18 +1,24 @@
+import { logger } from 'lib/logger'
+const moduleLogger = logger.child({ namespace: ['consoleArt'] })
```

Comment:
revert and ignore this file - can't render the console art nicely

<img width="960" alt="image" src="https://user-images.githubusercontent.com/88504456/187001974-c23d5e03-5bcb-40a2-9374-4f531a47f236.png">

Reply:
Nice catch ser - 1fc1bc338ea5a8ea8be2bbd41939542a92283300

<img width="458" alt="image" src="https://user-images.githubusercontent.com/17035424/187041016-97893e1a-3f7e-43a5-9ebd-b7ef5486940d.png">